### PR TITLE
feat(angular/schematics): set lean test configuration on ng-update migration

### PR DESCRIPTION
**Pull request by jeripeierSBB**

### CODE REVIEW

File: src/angular/schematics/ng-add/clean-up.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ import {
 
 import { DuplicateMigration } from './migrations/duplicate-migration';
 
+// noinspection JSUnusedGlobalSymbols
```
Comment (kyubisation) [inline src/angular/schematics/ng-add/clean-up.ts:10]: Why is this necessary?

**Review by kyubisation — APPROVED** [review]
LGTM 👍

File: src/angular/schematics/ng-add/clean-up.ts (hunk 1 above)
Comment (jeripeierSBB) [reply src/angular/schematics/ng-add/clean-up.ts:10]: Intellij warns me about unused function

Comment (kyubisation) [reply src/angular/schematics/ng-add/clean-up.ts:10]: Ok 👍
